src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+import service.FruitShopService;
+import service.impl.FruitShopServiceImpl;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final FruitShopService fruitShopService = new FruitShopServiceImpl();
+
+    public static void main(String[] args) {
+        fruitShopService.processData("inputDateFile.csv", "file.csv");
```

Comment:
invalid output file location
https://github.com/mate-academy/jv-fruit-shop/blob/master/checklist.md#place-the-input-and-output-files-into-the-srcmainresources-folder

Reply:
also, move them directly to the FruitShopService. less arguments we have - more convenient will be to use our code